src/libraries/ConstantProductOracleMath.sol

```diff
@@ -38,7 +38,6 @@ library ConstantProductOracleMath {
         spotPrice = reserve1.divWadUp(reserve0);
 
         int256 rawResult = LogCompression.toLowResLog(spotPrice);
-        assert(rawResult >= type(int112).min && rawResult <= type(int112).max);
```

Comment:
removed these as the output from `LogCompression.toLowResLog` is guaranteed to be within range